pkg/internal/retry/retry.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/google/go-containerregistry/pkg/internal/retry/wait"
 )
 
+// Backoff is an alias for wait.Backoff.
```

Comment:
![giphy (2)](https://user-images.githubusercontent.com/210737/64535044-9541ba80-d2e4-11e9-818e-688d8646b648.gif)

If it's for backward-compatibility, users shouldn't be depending in `internal` packages anyway.

Reply:
I didn't know until I went to fix it, see comment.